test-clients/src/main/java/com/hedera/services/bdd/suites/perf/mixedops/MixedOpsLoadTest.java

```diff
@@ -259,7 +258,6 @@ protected HapiApiSpec runMixedOps() {
 												TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT,
 												INVALID_TOKEN_ID,
 												TRANSACTION_EXPIRED,
-												TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED,
```

Comment:
Can this ResponseCode be removed from protobufs ?

Reply:
we cannot I think, backward compatibility issues.